### ISSUE TITLE
关于特征提取网络替换

### ISSUE THREAD

**Issue by zcswdt** [open-mmlab/mmdetection#9107]
### Prerequisite

- [X] I have searched [the existing and past issues](https://github.com/open-mmlab/mmdetection/issues) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version](https://github.com/open-mmlab/mmdetection).

### 💬 Describe the reimplementation questions

您好我想使用Yolox作为目标检测，请问我可以替换特征提取网络darkertnet为swin-transformer做完特征提取网络吗？谢谢

### Environment

windows11,python

**Comment by ZwwWayne** [comment]
Yes, you can. But you need to be careful to set the config.  You only need to change the config of backbone and update some relevant settings of neck to do that.
